sc2reader/readers.py

```diff
@@ -210,7 +210,13 @@ def __call__(self, data, replay):
                         licenses=[
                             data.read_uint32()
                             for i in range(
-                                data.read_bits(13 if replay.base_build >= 70154 else 9)
+                                data.read_bits(
+                                    16
+                                    if replay.base_build >= 77379
+                                    else 13
+                                    if replay.base_build >= 70154
+                                    else 9
```

Comment:
A double ternary if is not really Pythonic.  What about:
```python
if replay.base_build >= 77379:
    read_bits = 16
elif replay.base_build >= 70154:
    read_bits = 13
else:
    read_bits = 9
data.read_bits(read_bits)
```

Reply:
It might not be Pythonic, but it's consistent with rest of the code. So I'd rather keep it as is.

Unfortunately decoding functions are already hardly readable. And if there's something to be done with it, it should be tackled in separate PR IMO.